packages/static/src/tabs/tabs-paneless.tsx

```diff
@@ -0,0 +1,31 @@
+import React, {Key} from 'react'
+import TabItem from './item'
+import Tabs, {TabsProps} from './tabs'
+interface ItemProps {
+  key: Key
+  title: string
+}
+
+type TProps = Omit<TabsProps, 'children'>
+
+interface Props extends TProps {
+  items: ItemProps[]
+}
+
+export type TabsPanelessProps = Props
```

Comment:
why would you need to reassign Props to TabsPanelessProps?

Reply:
This can be a form of data hiding and can help prevent accidental misuse of internal properties.